github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+|                    | varea-windows-server     |                  |                 |
+|                    |                          |                  |                 |
+
+Project - NA for now
+--------------------------------------------------------------------------
+
+| ***xxxx*** | ***xxx*** |
+|------------|-----------|
+| xxx        | xxx       |
+
+
+> **No labels for this area at this time.**
+
+Milestones
+----------------------------------------------------------------
+
+| ***Release (Iteration path)***                                                            |
+|-------------------------------------------------------------------------------------------|
+| 6.3, 6.4, etc. (define specifically once we get a build number at end of release)         
+                                                                                            
+ 6.3 CU1, 6.4 CU1, etc. (define specifically once we a get build number at end of release)  
+                                                                                            
+ > Example 6.3.xxx.xx                                                                       
+ >                                                                                          
+ > All work items update automatically once the milestone is updated. (= minimal effort)    
+                                                                                            
+ There is not a vNext or backlog release.                                                   
+                                                                                            
+ > No Milestone = backlog                                                                   |
+
+Happy Path By issue
+===================
+
+Path below based on current State/Phase Labels
+
+****Question Happy Path****
+
+*Open -&gt; triaged -&gt; needs-more-info (optional) -&gt; in-progress -&gt; Closed. *
+
+****Bug or Documentation Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; need-more-info (optional) -&gt; understood (optional) -&gt; in -progress -&gt; Closed. *
+
+****Enhancement Happy Path****
+
+*Open -&gt; triaged -&gt; investigating (optional) -&gt; design -&gt; in -progress -&gt; review -&gt; Closed. *
+
+> ****Questions, Defects and Doc issues****
+>
+> ***'Optional' Phases will apply to issues that are immediately understood allowing them to bypass many of the phases, moving the bug right to 'in-progress'.***
+
+****Enhancement Issues****
+
+> ***Where at 'review' the feature is reviewed for release and obtains any special sign-off needed from feature teams (if applicable). If not done (completed or has issues), go back to investigate (if needed) or move to design or in-progress for additional work. Repeat as needed. Once the feature can pass the review process (Whatever that may be for the given feature) it moves to closed***
+
+Issue Lifecycle
```

Comment:
I think focusing on the principle, that all issues should maintain variables describing their status (area, type, status) is more valuable and natural than attempting to capture it in a list of instructions.  It should be intuitive and you shouldn't need to follow any sort of runbook like this.